src/api/liskApi.js

```diff
@@ -12,212 +12,176 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { LIVE_PORT, TEST_PORT, GET, POST } from 'constants';
+import {
+	LIVE_PORT,
+	SSL_PORT,
+	TEST_PORT,
+	TESTNET_NETHASH,
+	MAINNET_NETHASH,
+} from 'constants';
 import config from '../../config.json';
-import * as privateApi from './privateApi';
-import * as utils from './utils';
+import * as resources from './resources';
+
+const commonHeaders = {
+	'Content-Type': 'application/json',
+	os: 'lisk-js-api',
+	version: '1.0.0',
+	minVersion: '>=0.5.0',
+};
 
 export default class LiskAPI {
 	constructor(providedOptions) {
 		const options = Object.assign({}, config.options, providedOptions);
```

Comment:
This needs to change to `Object.assign({ headers: {} }, config.options, providedOptions);` otherwise the code below can error.

Reply:
do we not rely on `config.json`?
if we set the first object to `{ headers: {}}`, then we need to delete `headers` option from `config.options`

Reply:
Oh sorry! Didn't realise there was already a `headers` object in `config.options`. Ah, then maybe we do need the nested check lower down in case someone changes that to `undefined` or something.